roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -59,6 +59,16 @@
   failed_when: "'already exists' not in osmiq_perm_task.stderr and osmiq_perm_task.rc != 0"
   changed_when: osmiq_perm_task.rc == 0
 
+- name: Configure 3_2 role/user permissions
+  command: >
+    {{ openshift.common.admin_binary }} {{item}}
+    --config={{manage_iq_tmp_conf}}
+  with_items: "{{manage_iq_openshift_3_2_tasks}}"
+  register: osmiq_perm_3_2_task
+  failed_when: osmiq_perm_3_2_task.rc != 0
+  changed_when: osmiq_perm_3_2_task.rc == 0
```

Comment:
you might be able to use `oc get clusterpolicybindings system:image-auditor` with the `--template` argument to return just the user field. I would have to defer to @abutcher or @deads2k on the syntax for that though.

You might also want to consider passing `-o yaml` or `-o json` and then you could use the `from_yaml` or `from_json` filter to better interact with the return value as well.